scripts/playout_controls.sh

```diff
@@ -994,6 +995,10 @@ case $COMMAND in
         sudo rm WifiIp.mp3
         /usr/bin/php /home/pi/RPi-Jukebox-RFID/scripts/helperscripts/cli_ReadWifiIp.php
         ;;
+    bluetoothtoggle)
+        if [ "${DEBUG_playout_controls_sh}" == "TRUE" ]; then echo "   ${COMMAND}" >> ${PATHDATA}/../logs/debug.log; fi
+	$PATHDATA/../components/bluetooth-sink-switch/bt-sink-switch.py $VALUE
```

Comment:
@ChisSoc 
Hi Chris, you really put a lot of thought and work into this!!! Kudos.
Looking at the code in github, it seems that here is an incoherence with the indents. Is that possible? 
Is this code you have running?

Reply:
Hi Micz,

thank you very much! I was actually worried, I had put the new feature too deeply into the code base. Glad you like it! 

And also thanks for pointing out my misspelled user name! Actually, it is Chris, and should have been ChrisSoc. A lot of work had been done late at night :-) Solves the mystery why I only could log in with my e-mail.

This has most likely been said before, but cannot be said often enough: This is a very inspiring project! I finished my Box in May 2020 but with Corona and all the implementation of this feature got delayed and the work to bring this feature up to integration quality even more though. I will post pictures soon.

Now, to the issue: The indentation was due to a tabbed indent instead of space characters. I need to apologize for this slip up. I was probably due to me coding some of this directly via SSH on the RPi with Emacs only partially setup to my usual configuration. It is fixed! Bye the way: the code was running on my two RPis - propably because it is a Shell script and not Python? It is working as expected with the fix.

~~~
$ ./playout_controls.sh -c=bluetoothtoggle -v=toggle
No bluetooth device connected. Defaulting to "Output 1".
Switched audio sink to "Output 1"
~~~